docs/changelog/89112.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 89112
+summary: Make param check when create custom analyzer
+area: Search/Analysis
+type: enhancement
+issues:
+ - 85710
```

Comment:
the changelog generation is automatic in most situations, no need to create the entry yourself. And it gets updated as the labels and title of the PR get updated. Could you remove the changelog from your PR, then it should get automatically created with the expected area and all other fields. The current build errors are around the wrong area.